contracts/seed/Seed.sol

```diff
@@ -279,13 +312,22 @@ contract Seed {
     */
     function close() public onlyAdmin isActive {
         // transfer seed tokens back to admin
-        require(
-            seedToken.transfer(admin, seedToken.balanceOf(address(this))),
-            "Seed: should transfer seed tokens to admin"
-        );
-
-        closed = true;
-        paused = false;
+        if(minimumReached){
+            // amount of seed tokens to be distributed = seedAmountAtStart - seedRemainder
+            uint256 seedToTransfer = seedAmountAtStart.sub(seedAmountAtStart.sub(seedRemainder));
```

Comment:
why not use seedClaimed?  In any case, this needs to also include the fees.

Reply:
Yes, this changes have been made, will be in the new PR.

Reply:
we can't use ```seedClaimed```, because if the admin try to close before investors have claimed it. So we can simply use ```seedRemainder```.